src/main/java/net/sf/jabref/model/entry/SpecialFields.java

```diff
@@ -0,0 +1,43 @@
+package net.sf.jabref.model.entry;
+
+
+public class SpecialFields {
+
+    public static final String FIELDNAME_PRINTED = "printed";
+    public static final String FIELDNAME_PRIORITY = "priority";
+    public static final String FIELDNAME_QUALITY = "qualityassured";
+    public static final String FIELDNAME_RANKING = "ranking";
+    public static final String FIELDNAME_READ = "readstatus";
+    public static final String FIELDNAME_RELEVANCE = "relevance";
+
+    public static final String PREF_SPECIALFIELDSENABLED = "specialFieldsEnabled";
```

Comment:
Yes and I argued a lot with Simon a while ago about this since I was the opinion, that all the smaller preference classes should contain their keys and default values. In the end Simon convinced me that for now it is better if everything stays in one place and we will later reconsider moving it to different places.